packages/app-mobile/components/voiceTyping/VoiceTypingDialog.tsx

```diff
@@ -142,22 +168,44 @@ const VoiceTypingDialog: React.FC<Props> = props => {
 		return <Text variant='labelSmall'>{preview}</Text>;
 	};
 
-	const headerAndStatus = <Text variant='bodyMedium'>{`${_('Voice typing...')}\n${renderContent()}`}</Text>;
 	return (
-		<Banner
-			visible={true}
-			icon={renderIcon()}
-			actions={[
-				{
-					label: _('Done'),
-					onPress: onDismiss,
-				},
-			]}
-		>
-			{headerAndStatus}
-			<Text>{'\n'}</Text>
-			{renderPreview()}
-		</Banner>
+		<Surface>
```

Comment:
This pull request switches from a `<Banner>` to a `<Surface>`. React Native Paper sets `role=alert` and `accessibilityLiveRegion=polite` on the content of `<Banner>`s. This caused TalkBack to re-read "Voice typing... Please record your voice... [[recognised text here]]" when new text was recognised.